docs/welcome.md

```diff
@@ -4,7 +4,7 @@ title: Welcome to Hubs
 sidebar_label: Welcome
 ---
 
-**Important Notice:** Mozilla Hubs will shut down on May 31, 2024. For more information about the shutdown and how you can support Hubs' life beyond Mozilla, please read this article: https://hubs.mozilla.com/labs/sunset/
+<!-- **Important Notice:** Mozilla Hubs will shut down on May 31, 2024. For more information about the shutdown and how you can support Hubs' life beyond Mozilla, please read this article: https://hubs.mozilla.com/labs/sunset/ -->
```

Comment:
I don't think this shutdown notice needs to be preserved, and Heather has a PR open to remove it.  Maybe we could merge Heather's PR instead?

Reply:
However it gets removed seems good to me.